util/linuxfw/nftables_runner.go

```diff
@@ -107,6 +107,12 @@ func (n *nftablesRunner) AddDNATRule(origDst netip.Addr, dst netip.Addr) error {
 	if err != nil {
 		return err
 	}
+	rule := dnatRuleForChain(nat, preroutingCh, origDst, dst, nil)
```

Comment:
Added a new helper to create a rule as we use the same logic in one of the new methods